<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPane</class>
 <widget class="QWidget" name="ControlPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>515</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>641</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageStdCtrl">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>标准控制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>111</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0" rowspan="3">
        <widget class="QSlider" name="sldGo">
         <property name="minimum">
          <number>-5</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignTop">
        <widget class="QPushButton" name="btnForward">
         <property name="text">
          <string>前进</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnGoZero">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignBottom">
        <widget class="QPushButton" name="btnBackward">
         <property name="text">
          <string>后退</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>261</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0" colspan="3">
        <widget class="QSlider" name="sldOffet">
         <property name="minimum">
          <number>-5</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="btnOffsetLeft">
         <property name="text">
          <string>左移</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnOffsetZero">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnOffertRight">
         <property name="text">
          <string>右移</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>261</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0" colspan="3">
        <widget class="QSlider" name="sldTurn">
         <property name="minimum">
          <number>-5</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="btnTurnLeft">
         <property name="text">
          <string>左转</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnTurnZero">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnTurnRight">
         <property name="text">
          <string>右转</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="title">
       <string>手动</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>20</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="title">
       <string>自动</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_18">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnThrsSwitch">
         <property name="text">
          <string>推进器开关</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnCamSwitch">
         <property name="text">
          <string>摄像头开关</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>151</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_21">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_19">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnDepthClosedLoop">
           <property name="text">
            <string>深度闭环</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="depthVal"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnRefreshDepth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>更新深度</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_20">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnCourseClosedLoop">
           <property name="text">
            <string>航向闭环</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="courseVal"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnRefreshCourse">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>更新航向</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pageAdvCtrl">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>高级控制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>任务控制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageParaSet">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>参数设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>571</width>
     <height>185</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="3">
         <widget class="QPushButton" name="btnParaSet">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>参数设置</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnStdCtrl">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>标准控制</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnAdvCtrl">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>高级控制</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnTaskCtrl">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>任务控制</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="txtDate">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="txtTime">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;已运行时间：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="txtElapsedTime">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0m - 0s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_21">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>…</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>深度：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="txtYaw">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>偏航：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="txtPit">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>横滚：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="txtRol">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>俯仰：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_5">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_15">
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="line_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
